src/irmin-pack/unix/file_manager.ml

```diff
@@ -519,11 +520,11 @@ struct
             | `Unknown_major_pack_version _ ) as e ->
             e)
 
-  let swap t ~generation ~unlink =
+  let swap t ~generation ~unlink ~last_copied_offset =
```

Comment:
It isn't really the offset of the last copied byte, it's one after that. What about just `end_offset`?